### ISSUE TITLE
Incompatible with logging

### ISSUE THREAD

**Issue by BenjaminHoegh** [twill-tools/twill#10]
Found a funny bug, if you import `twill.commands` the python module `logging` can't access any files and will print all calls in the console instead.

Python version: 3.10.2
Twill version: 3.0.1

Here is a code there gets hit by the problem
```python
from twill.commands import *
import logging

logging.basicConfig(
    level=logging.DEBUG,
    format="{asctime} {levelname:<8} {message}",
    datefmt='%Y-%m-%d %H:%M:%S',
    style='{',
    filename='log.log',
    filemode='a'
)

logging.critical('This is a critical level msg!')
logging.error('This is a error level msg!')
logging.warning('This is a warning level msg!')
logging.info('This is a info level msg!')
logging.debug('This is a debug level msg!')
```

**Comment by Cito** [comment]
Hm, yes, I think the problem is that `getLogger()` is called without a name [here](https://github.com/twill-tools/twill/blob/3cd700ad778362615b4896fcb1a875469bec94c9/twill/__init__.py#L51).

Is this fixed when you replace this with `getLogger('twill')`?

**Comment by BenjaminHoegh** [comment]
Yes if you replace `getLogger()` with `getLogger('twill')` in twills ´__init__py´ it works

**Comment by Cito** [comment]
I have released a fixed version 3.0.2.

**Comment by BenjaminHoegh** [comment]
Thank you so much, you the best :D
